misc/python/materialize/scalability/endpoints.py

```diff
@@ -77,12 +78,16 @@ def up(self) -> None:
             self.composition.up("postgres")
             self._port = self.composition.default_port("postgres")
 
+    def __str__(self) -> str:
+        return "postgres"
```

Comment:
Hm, this is also interesting that it overrides the inherited mz version. We could introduce a further base class `MzEndpoint` but ok if we don't want to do this just because of this.

Reply:
Do we care about the PG version?

Reply:
No, I wanted to do a few simple postgres/mz comparisons , so the postgres version right now is not important.